_data/projects/.net-compiler-platform-roslyn.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: .NET Compiler Platform ("Roslyn")
+desc: The .NET Compiler Platform ("Roslyn") provides open-source C# and Visual Basic
+  compilers with rich code analysis APIs.
+site: https://github.com/dotnet/roslyn
+tags:
+- roslyn
+- csx
+- script
+- compiler
+- C#
+- VB.NET
```

Comment:
TIL what these are called:

>  **Block sequences** indicate each entry with a dash and space ( “- ”).
> YAML also has **flow styles**, using explicit indicators rather than indentation to denote scope. The flow sequence is written as a comma separated list within square brackets.